toolchain/install/install_paths.h

```diff
@@ -82,6 +82,9 @@ class InstallPaths {
   // using Carbon in an environment with an unusual path to the installed files.
   static auto Make(llvm::StringRef install_prefix) -> InstallPaths;
 
+  // Constructor only used in errors or globals prior to initialization.
+  InstallPaths() { SetError("No prefix provided!"); }
```

Comment:
Should this be set up as constant initialization?
https://google.github.io/styleguide/cppguide.html#Static_and_Global_Variables

Reply:
Yeah, I was not totally satisfied with this. But I think actually your suggestion of a pointer is even better, I'm going to switch to that I think...